vault/resource_consul_secret_backend_role.go

```diff
@@ -88,6 +117,7 @@ func consulSecretBackendRoleResource() *schema.Resource {
 				Optional:    true,
 				Description: "Specifies the type of token to create when using this role. Valid values are \"client\" or \"management\".",
 				Default:     "client",
+				Deprecated:  "Deprecated in Vault as of 1.11",
```

Comment:
The `token_type` field isn't strictly ignored. Even when using Vault 1.11, if the version of Consul used is very old (pre-1.4), then the field will still be needed. It was deprecated in Vault because the most recent versions of Consul completely removed the old ACL system which is the only place where `token_type` gets used.

Would you still prefer purely documentation over a deprecation warning for this?